azure-devops/azext_devops/dev/common/identities.py

```diff
@@ -110,5 +110,11 @@ def get_display_name_from_identity(identity):
     return identity.provider_display_name
 
 
+def get_account_from_identity(identity):
```

Comment:
Should  we have some test for this. If the strings in the API change we will not catch it.

Reply:
I'll take this up in some another PR. Probably would add tests for another functions as well.